packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx

```diff
@@ -20,11 +20,13 @@ export const DefaultHeader = ({
   bodyText,
 }: DefaultHeaderProps): JSX.Element => {
   return (
-    <View flex="1">
-      <View color="font.primary" fontWeight="bold">
+    <View className={`${LivenessClassNames.StartScreenHeader}`}>
+      <View className={`${LivenessClassNames.StartScreenHeader}__heading`}>
```

Comment:
for consistency, shouldn't __heading and __body be their own classnames as well?

Reply:
do you mean like we should have a variable for them? We already have separate classes for them. This is just the pattern i saw we were already using in these files.

Reply:
Yeah, why not creating their own classnames?

Reply:
Yeah, I mean separate variables for them so they aren't being created in the file